apollo-openapi/src/main/java/com/ctrip/framework/apollo/openapi/client/service/ItemOpenApiService.java

```diff
@@ -189,4 +193,37 @@ public void removeItem(String appId, String env, String clusterName, String name
     }
 
   }
+
+  @Override
+  public OpenPageDTO<OpenItemDTO> findItemsByNamespace(String appId, String env, String clusterName,
+                                                       String namespaceName, int page, int size) {
+    if (Strings.isNullOrEmpty(clusterName)) {
+      clusterName = ConfigConsts.CLUSTER_NAME_DEFAULT;
+    }
+    if (Strings.isNullOrEmpty(namespaceName)) {
+      namespaceName = ConfigConsts.NAMESPACE_APPLICATION;
+    }
+
+    checkNotEmpty(appId, "App id");
+    checkNotEmpty(env, "Env");
+    checkPage(page);
+    checkSize(size);
+
+    OpenApiPathBuilder pathBuilder = OpenApiPathBuilder.newBuilder()
+            .envsPathVal(env)
+            .appsPathVal(appId)
+            .clustersPathVal(clusterName)
+            .namespacesPathVal(namespaceName)
+            .itemsPathVal("")
+            .addParam("page", page)
+            .addParam("size", size);
+
+    try (CloseableHttpResponse response = get(pathBuilder)) {
+      Type type = new TypeToken<OpenPageDTO<OpenItemDTO>>(){}.getType();
```

Comment:
How about make this local variable to a private static java field(maybe uses static nested classes for lazy load)? To avoid allocation new instance when use http get everytime.